.github/workflows/backward-compat-tests.yml

```diff
@@ -0,0 +1,58 @@
+#
```

Comment:
Kindly also see to it that these tests are not running in the catch all tests.

Reply:
good observation. I checked it and they run only in this job, thanks